examples/cpp/function-execution/main.cpp

```diff
@@ -38,147 +34,114 @@
 #include <geode/FunctionService.hpp>
 #include <geode/CacheableString.hpp>
 
-// Use the "geode" namespace.
-using namespace apache::geode::client;
+using apache::geode::client::Cache;
+using apache::geode::client::CacheableArrayList;
+using apache::geode::client::CacheableKey;
+using apache::geode::client::CacheableString;
+using apache::geode::client::CacheableVector;
+using apache::geode::client::CacheFactory;
+using apache::geode::client::Exception;
+using apache::geode::client::FunctionService;
+using apache::geode::client::Region;
+using apache::geode::client::RegionShortcut;
+
+const auto getFuncIName = std::string("ExampleMultiGetFunction");
+
+const int EXAMPLE_ITEM_COUNT  = 6;
```

Comment:
Does this still have a place now that we have fixed set of keys and values?

Reply:
Nope - removed.